lib/index.js

```diff
@@ -74,12 +78,6 @@ Appboy.prototype.loaded = function() {
  */
 
 Appboy.prototype.identify = function(identify) {
-  var userId = identify.userId();
-  // Appboy doesn't want you to call changeUser unless a userId is present because they do automatic anonymous user tracking on their end 
-  // https://www.appboy.com/documentation/Web/#setting-user-ids
-  if (!userId) return;
-  window.appboy.changeUser(userId);
```

Comment:
what if there is no `userId` during page load, and then without refreshing the page I call `.identify()`. That means `changeUser` will never be called. So this would effectively mean that appboy wouldn't know who the user is until _after_ they refresh. Is that still okay?